pkg/test/env/istio.go

```diff
@@ -54,8 +54,14 @@ var (
 	// nolint: golint, stylecheck
 	PULL_POLICY Variable = "PULL_POLICY"
 
+	// KUBECONFIG is the list of Kubernetes configuration files. If configuration files are specified on
+	// the command-line, that takes precedence.
+	// nolint: golint, stylecheck
+	KUBECONFIG Variable = "KUBECONFIG"
+
 	// ISTIO_TEST_KUBE_CONFIG is the Kubernetes configuration file to use for testing. If a configuration file
 	// is specified on the command-line, that takes precedence.
+	// TODO(nmittler): Remove this in favor of KUBECONFIG.
 	// nolint: golint, stylecheck
 	ISTIO_TEST_KUBE_CONFIG Variable = "ISTIO_TEST_KUBE_CONFIG"
```

Comment:
do we even set this? I didn't see it in our repo

Reply:
I'm not entirely sure. I guess if it's not in test-infra, we can remove.